README.md

```diff
@@ -72,6 +73,14 @@ Each task requires certain inputs which are to be specified in your `build.gradl
    - `timeout`: *Optional* - If specified, the task will wait for the risk score to be calculated. Default: 0 seconds
    - `maxRiskScore`: *Optional* - If specified, the task will fail if the risk score is higher than the specified value.
 
+#### analyzeProject
+
+- `url`: Dependency Track API URL
+- `apiKey`: Dependency Track API KEY
+- `projectUUID`: *Optional* - You need to set UUID or projectName and projectVersion
```

Comment:
Is the logic here `UUID || (projectName && projectVersion)` or is it `(UUID && projectVersion) || (projectName && projectVersion)`?

Reply:
It's "UUID || (projectName && projectVersion)"